node/node.go

```
@@ -869,19 +868,27 @@ func (n *Node) initInfoAPI() error {
 		n.Log.Info("skipping info API initialization because it has been disabled")
 		return nil
 	}
+
 	n.Log.Info("initializing info API")
+
+	primaryValidators, _ := n.vdrs.GetValidators(constants.PrimaryNetworkID)
```

We ignore the error here, but this could still create a problem.